packages/api/index.ts

```diff
@@ -16,3 +18,5 @@ export type RouterInputs = inferRouterInputs<AppRouter>;
  * @example type HelloOutput = RouterOutputs['example']['hello']
  **/
 export type RouterOutputs = inferRouterOutputs<AppRouter>;
+
+export const createCaller = createCallerFactory(appRouter);
```

Comment:
remove code related to caller and just extract notfiication logic into its own function